.github/workflows/build.yml

```
@@ -1,18 +1,30 @@
 name: Build
-on: [ push, pull_request ]
+on: [ push, pull_request, workflow_dispatch ]
 jobs:
   test:
     strategy:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest ]
-        ruby: [ '2.7', '3.0' ]
+        ruby: [ '3.3', '3.4' ]
     runs-on: ${{ matrix.os }}
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: ruby/setup-ruby@v1
+      - name: Check out repository
+        uses: actions/checkout@v2
```

Any reason we're using v2 here and v3 in the Gem Push workflow?

```suggestion
        uses: actions/checkout@v3
```

---

resolved in 04c3276